### PR TITLE
Setup Tailwind CSS

### PR DESCRIPTION
**Pull request by Yvad60**
## What does the PR Do
it installs and configure Tailwind CSS as and removes material UI

## Tasks done 
- [x] Tailwind is installed and well configured
- [x] Material UI is removed successfully
- [x] The landing page is styled using Tailwind CSS

## How to test? 
```
git checkout ch-setup-tailwind-css-cp-89
npm i or yarn 
npm start or yarn start
```

### CODE REVIEW

File: postcss.config.cjs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const tailwindcss = require('tailwindcss');
+const autoprefixer = require('autoprefixer');
+
+module.exports = {
+	plugins: [tailwindcss('./tailwind.config.cjs'), autoprefixer],
+};
```
Comment (Abdulkeza) [inline postcss.config.cjs:6]: leave one at the end.

File: postcss.config.cjs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const tailwindcss = require('tailwindcss');
+const autoprefixer = require('autoprefixer');
```
Comment (Abdulkeza) [inline postcss.config.cjs:2]: do we really need these variables?  like you are assigning values in the new variable which should be done by only one line. and u are consuming memories.

Remember the big-O notation.???

File: src/app.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (Abdulkeza) [inline src/app.css:3]: leave the space at the end

**Review by Abdulkeza — CHANGES_REQUESTED** [review]
Great job Yves, left some comments bellow.

File: postcss.config.cjs (hunk 2 above)
Comment (Yvad60) [reply postcss.config.cjs:2]: Hahaha! Actually, I wanted to use ES imports which add a variable too but it seems PostCSS doesn't allow it but I found that using require in the code block is not recommended especially by Eslint check [Here](https://eslint.org/docs/rules/global-require) 
That's why I added my require on the top

File: src/app.css (hunk 3 above)
Comment (Yvad60) [reply src/app.css:3]: but there is a line on my end I don't know the issue

**Review by Abdulkeza — APPROVED** [review]
Great job Yvad, now is ready to be merged...

**Pull request merged**